PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -424,6 +425,31 @@ def isFlavour(x,f):
                    jet.mcFlavour = 0
 
         self.heaviestQCDFlavour = 5 if len(self.bqObjects) else (4 if len(self.cqObjects) else 1);
+
+
+    def makeNIsr(self,event):
+
+        event.nIsr = 0
+
+        for jet in self.cleanJetsAll:
+
+            if jet.pt()<30.0: continue
+            if abs(jet.eta())>2.4: continue
+            matched = False
+            for mc in event.genParticles:
+                if matched: break
+                if (mc.status()!=23 or abs(mc.pdgId())>5): continue
+                momid = abs(mc.mother().pdgId())
+                if not (momid==6 or momid==23 or momid==24 or momid==25 or momid>1e6): continue
```

Comment:
@steggema , thanks for pointing this out. This implementation has only been designed to work for ttbar and SUSY signal models. I will therefore move it to a different, less generic, analyzer